src/expr/parser.rs

```diff
@@ -25,34 +47,46 @@ named!(i64_literal<i64>,
   )
 );
```

Comment:
As the compiler _warns_ you, this parser is now unused -- perhaps you should remove it?